app/controllers/api/auth/podcast_imports_controller.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+class Api::Auth::PodcastImportsController < Api::BaseController
+  include ApiAuthenticated
+
+  api_versions :v1
+
+  filter_resources_by :account_id, :series_id
+
+  filter_params :url
+
+  def after_create_resource(res)
+    res.try(:import)
```

Comment:
So the API create will result in a synchronous import running?  How fast can this run?  (I'm all for sync operations, by the way!)

Also, I'm not sure what happens when an error bubbles up to this after_create hook.  Do you still get a 201?